src/js/components/FormField/FormField.js

```diff
@@ -61,13 +72,24 @@ const isGrommetInput = (comp) =>
   (grommetInputNames.indexOf(comp.displayName) !== -1 ||
     grommetInputPadNames.indexOf(comp.displayName) !== -1);
 
+const getFocusStyle = (props) => {
+  if (
+    props.focus &&
+    props.shouldSkipFocus &&
+    props.theme.formField?.focus?.conatinerFocus === false
```

Comment:
```suggestion
    props.theme.formField?.focus?.containerFocus === false
```
typo -- double check other places that might be relying on this logic